.jenkins/common_job_properties.groovy

```diff
@@ -168,6 +174,13 @@ class common_job_properties {
       scm(scm_schedule)
     }
 
+    // Enable triggering oneshots of pull requests. Users can comment the trigger phrase
+    // speficied in the postcommit job and have the job run against their PR to run
+    // tests not in the presubmit suite for additional confidence.
+    if (trigger_phrase != '') {
+      setPullRequestBuildTrigger(context, 'Jenkins PostCommit', '--none--', trigger_phrase)
```

Comment:
How will users see the result, particularly if I want to trigger multiple? Should we use 'Jenkins PostCommit'?